aggregators/1.0/modules/ROOT/nav.adoc

```diff
@@ -10,5 +10,5 @@
 ** xref:aggregators-size-timeout.adoc[Size-Based Aggregator Timeout and Max Size Configuration]
 ** xref:aggregators-multiple-examples.adoc[Aggregators Module Multiple Examples Configuration]
 * xref:aggregators-xml-maven.adoc[Aggregators Module XML and Maven Support]
-* xref:aggregators-troubleshooting.adoc[Troubleshoot Aggregators Module]
+* xref:aggregators-troubleshooting.adoc[Troubleshooting Aggregators Module]
```

Comment:
IMHO, this should be "Troubleshooting the Aggregators Module." It sounds funny without "the," but I approved since it can be changed down the road

Reply:
Fixed